index.html

```diff
@@ -95,7 +95,12 @@ <h4 class="program-head" >Book Reading</h4>
                     <h4 class="program-head" >Short Plays</h4>
                     <p class="program-para" >Short plays that reveals personal stories and explore complex social issues.</p>
                 </div>
-            </div>
+                <div class="program-container">
+                    <img class="program-img" src="img/museum-art-tour-icon-cartoon-real-walk-vector.webp" alt="Gallery Tour">
+                    <h4 class="program-head" >Wine Testing</h4>
+                    <p class="program-para" >Test some of the best wines in Addis Like Gudar and Achechia.</p>
+                </div>
```

Comment:
- [ ] Excellent job implementing the changes requested by the previous reviewer, but the program section is missing. Could you please add it back or make sure that it displays? Because it was previously present. 🤓 🛠️